src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,25 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type TodoInfoProps = {
```

Comment:
Make it simple, just use `Props` name everywhere
```suggestion
type Props = {
```